code/modules/mob/say.dm

```diff
@@ -1,3 +1,17 @@
+var/list/special_channel_keys = list(
+	  "r" = "right ear",
+	  "l" = "left ear",
+	  "i" = "intercom",
+	  "h" = "department",
+	  "+" = "special",
+	  "w" = "whisper",
+	  "ê" = "right ear",
+	  "ä" = "left ear",
+	  "ø" = "intercom",
```

Comment:
what's with runes for radio keys?

Reply:
Preserved from the ancient Slavic texts I think. I copied them from the old key list.